README.md

```diff
@@ -640,12 +640,13 @@ See also [A comparison of operating systems written in Rust](https://github.com/
   * [cargo-commander](https://crates.io/crates/cargo-commander) - A subcommand for `cargo` to run CLI commands similar to how the scripts section in `package.json` works [![Build and test](https://github.com/simonhyll/cargo-commander/actions/workflows/build.yml/badge.svg)](https://github.com/simonhyll/cargo-commander/actions/workflows/build.yml)
   * [cargo-count](https://crates.io/crates/cargo-count) - lists source code counts and details about cargo projects, including unsafe statistics
   * [cargo-deb](https://crates.io/crates/cargo-deb) - Generates binary Debian packages
-  * [cargo-deps](https://crates.io/crates/cargo-deps) - build dependency graphs
+  * [cargo-depgraph](https://crates.io/crates/cargo-depgraph) - Creates dependency graphs for cargo projects using cargo metadata and graphviz
+  * [cargo-deps](https://crates.io/crates/cargo-deps) - build dependency graphs. Unmaintained, see `cargo-depgraph`
```

Comment:
Given this appears to be very dead (not even seeing the repo anymore), can you remove this entirely please?